### PR TITLE
Add drop down menu to app top bar 

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
image

## Related Issue

- Fixes https://github.com/owncloud/web/issues/11019

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by JammingBen — APPROVED** [review]
LGTM and works nicely 👍

**Comment by pascalwengerter** [comment]
Nice seeing this come to live!

File: changelog/unreleased/enhancement-add-action-drop-down-to-app-top-bar [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: add action drop down top app top bar
```
Comment (kulmann) [inline changelog/unreleased/enhancement-add-action-drop-down-to-app-top-bar:1]: nobody noticed the typo 😓 😅

Comment (AlexAndBear) [reply changelog/unreleased/enhancement-add-action-drop-down-to-app-top-bar:1]: Oh no :( gonna fix this asap as possible !

Comment (kulmann) [reply changelog/unreleased/enhancement-add-action-drop-down-to-app-top-bar:1]: Beste!

**Review by kulmann — COMMENTED** [review]
Too late too approve, but man I love this 🥇

**Comment by AlexAndBear** [comment]
> Too late too approve, but man I love this 🥇 

Toodalo! 🥰

**Pull request merged**